sys-cluster/charliecloud/charliecloud-0.9.10.ebuild

```diff
@@ -0,0 +1,62 @@
+# Copyright 1999-2019 Gentoo Authors
+# Distributed under the terms of the GNU General Public License v2
+
+EAPI=7
+
+PYTHON_COMPAT=( python3_{5,6} )
+
+inherit python-single-r1
+
+if [[ ${PV} == "9999" ]] ; then
+	inherit git-r3
+	EGIT_REPO_URI="https://github.com/hpc/${PN}.git"
+	S="${WORKDIR}/${P}"
+else
+	SRC_URI="https://github.com/hpc/${PN}/archive/v${PV}.tar.gz -> ${P}.tar.gz"
+	KEYWORDS="~amd64 ~x86 ~amd64-linux ~x86-linux"
+fi
+
+DESCRIPTION="Lightweight user-defined software stacks for high-performance computing"
+HOMEPAGE="https://hpc.github.io/charliecloud/"
+
+SLOT="0"
+LICENSE="Apache-2.0"
+IUSE="doc examples +pv"
+
+# Extensive test suite exists, but downloads container images
+# directly and via Docker and installs packages inside using apt/yum.
+# Additionally, clashes with portage namespacing and sandbox.
+RESTRICT="test"
+
+RDEPEND="${PYTHON_DEPS}
+	=dev-lang/python-3*
+	pv? ( sys-apps/pv )
+"
+DEPEND="doc? (
+		dev-python/sphinx
```

Comment:
Add PYTHON_USEDEP for sphinx and the theme. I'm pretty sure they need to be installed for matching Python version for this to work.

Reply:
They are only used to generate documentation at compile time, and not at runtime. 
I tested:
```
PYTHON_TARGETS="python3_7" PYTHON_SINGLE_TARGET="python3_7" emerge -1av =charliecloud-9999
```
just now, having both `sphinx` and the theme only installed for 3.5 and 3.6, and the emerge went through without issues. So `PYTHON_USEDEP` should not be needed for these.

Reply:
Now try installing sphinx for 3.5 only, and theme for 3.6 only.

Reply:
Ah, now I understand! 
Yes, both need to match - they don't need to match the charliecloud python version, but they need to match each other. Ok, I'll add `PYTHON_USEDEP` of course to have this covered.